src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1071,191 +1048,218 @@ k
-            } else if (aircraft.trend < 0) {
-                alt_trend_char = String.fromCodePoint(0x1F853); // down arrow (symbola font)
+            // Draw ILS Clearance Indicator
+            cc.translate(-width2 - bar_width2, 0);
+            cc.lineWidth = bar_width;
+            cc.strokeStyle = red;
+            cc.beginPath(); // top arc start
+            cc.arc(0, -lock_offset, lock_size / 2, -pi_slice, pi + pi_slice, true);
+            cc.moveTo(0, -lock_size / 2);
+            cc.lineTo(0, -height2);
+            cc.stroke(); // top arc end
+            cc.beginPath(); // bottom arc start
+            cc.arc(0, lock_offset, lock_size / 2, pi_slice, pi - pi_slice);
+            cc.moveTo(0, lock_size - bar_width);
+            cc.lineTo(0, height2);
+            cc.stroke();  // bottom arc end
+
+            if (aircraft.isEstablishedOnCourse()) {
+                // Localizer Capture Indicator
+                cc.fillStyle = white;
+                cc.beginPath();
+                cc.arc(0, 0, lock_size / 5, 0, pi * 2);
+                cc.fill(); // Draw Localizer Capture Dot
             }
 
-            // Draw full datablock text
-            cc.textAlign = 'left';
-            cc.fillText(row1text, -width2 + paddingLR, -gap / 2 - lineheight);
-            cc.fillText(row2text, -width2 + paddingLR, gap / 2 + lineheight);
-            // Draw climb/level/descend symbol
-            cc.font = '10px symbola'; // change font to the one with extended unicode characters
-            cc.textAlign = 'center';
-            cc.fillText(alt_trend_char, -width2 + paddingLR + 20.2, gap / 2 + lineheight - 0.25);
-            cc.font = BASE_CANVAS_FONT;  // change back to normal font
+            cc.translate(width2 + bar_width2, 0);
+            // unclear how this works...
+            cc.beginPath(); // if removed, white lines appear on top of bottom half of lock case
+            cc.stroke(); // if removed, white lines appear on top of bottom half of lock case
 
             cc.restore();
         }
+
+        // Text
+        const gap = 3;          // height of TOTAL vertical space between the rows (0 for touching)
+        const lineheight = 4.5; // height of text row (used for spacing basis)
+        const row1text = cs;
+        const aircraftAltitude = round(aircraft.altitude * 0.01);
+        const aircraftSpeed = round(aircraft.groundSpeed * 0.1);
+        const row2text = `${leftPad(aircraftAltitude, 3)} ${leftPad(aircraftSpeed, 2)}`;
+
+        // TODO: remove the if/else in favor of an initial assignment, and update with if condition
+        if (aircraft.inside_ctr) {
+            cc.fillStyle = this.theme.DATA_BLOCK.TEXT_IN_RANGE;
+        } else {
+            cc.fillStyle = this.theme.DATA_BLOCK.TEXT_OUT_OF_RANGE;
+        }
+
+        if (aircraft.trend === 0) {
+            // small dash (symbola font)
+            alt_trend_char = String.fromCodePoint(0x2011);
```

Comment:
to the best of my knowledge, this is the only place we are using that font.  do you think we could remove this in favor of a smaller font?  Is that even possible with the characters we're using here?